### ISSUE TITLE
Carbon Calculator: Accessibility - Ensure every ARIA progressbar node has an accessible name

### ISSUE THREAD

**Issue by BrianGibbVS** [visitscotland/vs-component-library#245]
### Raised by
Brian Gibb

### Describe the bug
In the Carbon Calculator, each node in the progress bar needs to have an an accessible name.

Attached are examples of where this fails an accessibility scan.

### To Reproduce

1. Go to Carbon Calculator page
2. Conduct an Accessibility Scan on each of the Carbon Calculator steps.
3. See screenshot.

### Expected behavior
Each node should have an accessible name using an aria-label ot title attribute

### Page URL
https://www.visitscotland.com/travel-planning/responsible-tourism/carbon-calculator

### Environment
All

### Browser
All

### Device and version
N/A

### Impact on end user
Screen reader users are not able to discern the purpose of elements with role="progressbar" that do not have an accessible name.

### Impact on product development
N/A

### Additional context
N/A
